src/components/CommentsList.jsx

```diff
@@ -0,0 +1,29 @@
+import { useEffect, useState } from "react";
+import { fetchCommentsByArticleId } from "../utils/api";
+import CommentLine from "./CommentLine";
+
+const CommentsList = ({ article_id }) => {
+  const [comments, setComments] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    if (!article_id) return;
+
+    setIsLoading(true);
+    fetchCommentsByArticleId(article_id).then((comments) => {
+      setIsLoading(false);
+      setComments(comments);
+    });
+  }, [article_id]);
+  return isLoading ? (
+    <p>Loading...</p>
+  ) : (
+    <ul className="comments__container">
+      {comments.map((comment) => {
+        return <CommentLine key={comment.comment_id} comment={comment} />;
+      })}
```

Comment:
It's really great that you have a loading state here, but what would happen if there was an article with no comments? It would be nice if some feedback could be given to the user in this situation 👍